tests/conftest.py

```diff
@@ -11,6 +11,8 @@ def pytest_addoption(parser: Parser):
     """Add options to pytest parser."""
     parser.addoption("--path", action="store")
     parser.addoption("--token", action="store")
+    # A comma separated GitHub Personal Access Tokens, to help reduce rate limiting.
+    parser.addoption("--secondary-tokens", action="store")
```

Comment:
What about making the token optionally a comma separated list so that we could use `N` tokens? Just split it on `,` or something (not sure if that could be in the token) or something and that will mean that if 1 or more tokens can be supplied

Reply:
Yup, it currently can receive from 0 to N tokens :D
```python
tokens = [token.strip() for token in (secondary_tokens.split(",") if secondary_tokens else [])]
```

Reply:
Added another option `--tokens` so that we don't break tests in other PRs

Reply:
`--token` option is now optinally csv.